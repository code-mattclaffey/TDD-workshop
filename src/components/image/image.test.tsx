import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { Image } from '.';

afterEach(() => {
    cleanup();
});

describe('<Image />', () => {
    it('should match snapshot', () => {
        const { container } = render(
            <Image
                additionalClassNames="c-img"
                src="https://via.placeholder.com/400x300"
                alt="A placeholder image"
            />,
        );

        expect(container).toMatchSnapshot();
    });
});
