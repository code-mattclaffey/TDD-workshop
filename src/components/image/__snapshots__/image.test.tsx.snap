// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image /> should match snapshot 1`] = `
<div>
  <img
    alt="A placeholder image"
    class="c-img"
    src="https://via.placeholder.com/400x300"
  />
</div>
`;
